tests/retryReads-move-sharded.yaml

```diff
@@ -0,0 +1,62 @@
+initialConfiguration:
```

Comment:
Is there a particular reason some files in this PR use `.yml` but others use `.yaml`? AFAIK, mongodb/specifications uses `.yml` exclusively so perhaps we can stick to that for consistency.

Reply:
I checked this and historically the scenarios used .yaml while the validator files used .yml, so this PR continues with that split. I can change scenarios to be .yml if wanted.

Reply:
Actually my assessment was not quite correct. The existing scenario files used .yaml. The validator didn't use a file but specified the contents inline. When I created the files I used .yml extension creating the current inconsistency. So I can either change the added .yml files to .yaml to be consistent with what this repository has historically used or I can change the .yaml files to .yml to be consistent with specs repo.

Reply:
I'd vote for using `*.yml` everywhere to be consistent with our specs repo. @prashantmital: any objections?

Sorry if this comes off as bikeshedding, but I know in the specs repo we do refer to `*.yml` in scripts. I was thinking this could become an issue if https://github.com/mongodb-labs/drivers-atlas-testing/issues/100 were implemented down the line.

Reply:
I favor consistency as well. Let's use `*.yml`.